src/Sulu/Bundle/TagBundle/Entity/Tag.php

```diff
@@ -90,7 +84,7 @@ public function setChanged(\DateTimeImmutable $changed): self
         return $this;
     }
 
-    public function __toString()
+    public function __toString(): string
     {
         return (string) $this->getName();
```

Comment:
Since getName is typed to a string, we don't need to cast here.

Reply:
would keep as before empty string, doctrine fallbacks to a object hash I think then like when no __toString exists which is better in the trace.